python/python/glide/async_commands/cluster_commands.py

```diff
@@ -340,3 +340,30 @@ async def time(self, route: Optional[Route] = None) -> TClusterResponse[List[str
             TClusterResponse[List[str]],
             await self._execute_command(RequestType.Time, [], route),
         )
+
+    async def lastsave(self, route: Optional[Route] = None) -> TClusterResponse[int]:
+        """
+        Returns the Unix time of the last DB save timestamp or startup timestamp if no save was made since then.
+
+        See https://valkey.io/commands/lastsave for more details.
+
+        Args:
+            route (Optional[Route]): The command will be routed to a random node, unless `route` is provided,
+                in which case the client will route the command to the nodes defined by `route`.
+
+        Returns:
+            TClusterResponse[int]: The Unix time of the last successful DB save.
+                If a single node route is requested, returns an int representing the Unix time of the last successful DB save.
```

Comment:
Maybe add this (up to you)
```suggestion
                If not route or a single node route is requested, returns an int representing the Unix time of the last successful DB save.
```